src/initializers/bull/bull.ts

```diff
@@ -68,46 +100,6 @@ export default class Bull {
     const stats = this.queueNames.map(this.fetchStats, this);
     return Promise.all(stats);
   }
-
-  private recordMetric(key: string, value: number) {
-    if (newrelic) {
-      newrelic.recordMetric(key, value);
-    } else {
-      getLogger('bull.metrics').info(`${key}: ${value}`);
-    }
-  }
-
-  private async reportStats(queueNameCase?: string) {
-    const stats = await this.getStats();
-    const queueNameTransform = caseModifiers.includes(queueNameCase) ? _[queueNameCase] : _.camelCase;
-    stats.forEach(entry => {
-      this.recordMetric(`Bull/Queues/${queueNameTransform(entry.queue)}`, entry.count);
-      this.recordMetric(`Bull/QueuesFailed/${queueNameTransform(entry.queue)}`, entry.failed);
-    });
-  }
-
-  public startMetrics(cronExpression, queueNameCase?: string) {
```

Comment:
This is a breaking change, maybe keep the function and log a warning to make sure that apps that are using this via a cli won't break?